### ISSUE TITLE
Mac OS Installer

### ISSUE THREAD

**Issue by maujin111** [LibreSprite/LibreSprite#58]
As you know many graphic artists use Mac OS to do their work, either for paid work or as a hobby. A reddit user gave me this idea:

" _Might I suggest looking into using Homebrew to get it on the Mac. The platform may be able to get other library and package requirements in place w/out you needing to do too much custom work. There are a lot of straight Linux packages available in home brew that have little to no extra porting work done._ ". 

So, do you think it's worth giving it a try? Next week I could try to make an installer for mac. Wish me luck

**Comment by MintStudios** [comment]
Yeah, good idea. I'm not experienced with Mac, and I have never used homebrew before.

**Comment by jjhaggar** [comment]
Oh, that would be nice! Good luck! :D

I can test the binaries in an old Mac mini with OSX Yosemite 10.10.5 in case that it is of any help :)

**Comment by PerryHugh** [comment]
If you can get it working that would be fantastic. I don't have a Mac but attempted to build Skia locally and it failed due to authorization codes (Google's internal authorization codes... for a Google product) being declined. More recent updates can be built fine but breaks our dependency.

**Comment by DarkOleks** [comment]
I can test it on OSX High Sierra 10.13.6
